### PR TITLE
feat: add utility pallet

### PR DESCRIPTION
**Pull request by cuteolaf**

### CODE REVIEW

**Review by ndkazu — APPROVED** [review]

File: runtime/Cargo.toml [hunk 1]
```diff
@@ -141,4 +147,4 @@ try-runtime = [
 	"pallet-onboarding/try-runtime",
```
Comment (ndkazu) [inline runtime/Cargo.toml:147]: Is there a reason for not adding `"pallet-utility/try-runtime"` here?
and there is also the missing `"pallet-utility/runtime-benchmarks"` in the `runtime-benchmark` section.
Usually I refer to the following Files/folder to add pallets:
https://github.com/paritytech/substrate/blob/polkadot-v0.9.26/bin/node/runtime/Cargo.toml

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -595,6 +602,7 @@ construct_runtime!(
 		Preimage: pallet_preimage,
 		Council: pallet_collective::<Instance1>,
 		Democracy: pallet_democracy,
+		Utility: pallet_utility,
 		// flag add pallet runtime
 	}
 );
```
Comment (ndkazu) [inline runtime/src/lib.rs:608]: Here also, missing `[pallet_utility, Utility]` in the `define_benchmark` section.

**Review by ndkazu — CHANGES_REQUESTED** [review]

**Comment by letodunc** [comment]
To prevent missing things when adding a new pallet, there is a script that automatize it for you. It's **_add_new_pallet.sh_** in the scripts folder. You execute it from the root folder of the project. In your case, a use could be _**scripts/add_new_pallet.sh utility Utility**_. Everything will be correctly configured for the basic add in the project. Then you can customize it with what you need.

**Comment by ilhanu** [comment]
No sure why the coverage tests are failing, might have to do with the that it's an external PR

**Comment by ilhanu** [comment]
@cuteolaf can you please check the logs for the benchmarking and provide a fix.

**Comment by ndkazu** [comment]
A few things I do before sending a commit:
- `cargo check` to make sure that it compiles
- `cargo test` to be sure that all the tests are correctly configured and pass.
- `cargo build --release --features runtime-benchmarks` to be sure that all the benchmarks are correctly configured and pass.
sometimes ou will also have additionnal tests in your benchmarks (it is the case for some FairSquares pallets), in this case you also want to run the following:
- `cargo test --features runtime-benchmarks`

In my eperience, If all the checks above pass on your local node, your commit will checkout without ANY problem.

**Pull request merged**